docs/src/_guide/troubleshooting.md

```diff
@@ -30,7 +30,17 @@ If you have a question about Meltano, are having trouble getting it to work, or
     </div>
 </div>
 
-# Common Troubleshooting Issues
+# Common Issues
+
+*Problem: "Why do **incremental runs** produce duplicate data?"* 
+
+View this [Slack discussion](https://meltano.slack.com/archives/C013EKWA2Q1/p1659024288228009). Tl;dr: Singer takes an "at least once" approach to replication, so if you're encounting this, it might be intended behavior.
```

Comment:
Linking to Slack is unfriendly to newcomers who may not have (and not want) an account on the Meltano Slack.

Reply:
```suggestion
Singer takes an "at least once" approach to replication, so if you're encountering this, it might be intended behavior. [This issue](https://github.com/MeltanoLabs/Singer-Working-Group/issues/13) is a good summary of the current state and a proposal to change this behavior.
```